.github/workflows/functional-validation-publish.yml

```diff
@@ -0,0 +1,50 @@
+name: FunctionalValidation-PublishPackage
```

Comment:
Why is it a `FunctionalValidation`, not just `Validation`? IMO `FunctionalValidation` sounds like Functional/API Tests

Reply:
Haven't had any original idea for a library name. The point is that this validation library will use functional programming constructs from `CSharpFunctionalExtensions` (`Result<T>` or `Result<T,E>`).